pkg/virt-operator/resource/apply/reconcile.go

```diff
@@ -1,5 +1,5 @@
 /*
- * This file is part of the KubeVirt project
+r* This file is part of the KubeVirt project
```

Comment:
Probably should be here.

Reply:
oops, good catch, that's me trying (and failing) to exit vim.